src/Compilers/VisualBasic/Portable/CommandLine/CommandLineParser.vb

```diff
@@ -1137,7 +1137,8 @@ lVbRuntimePlus:
                     languageVersion:=languageVersion,
                     documentationMode:=If(parseDocumentationComments, DocumentationMode.Diagnose, DocumentationMode.None),
                     kind:=SourceCodeKind.Regular,
-                    preprocessorSymbols:=AddPredefinedPreprocessorSymbols(outputKind, defines.AsImmutableOrEmpty()))
+                    preprocessorSymbols:=AddPredefinedPreprocessorSymbols(outputKind, defines.AsImmutableOrEmpty()),
+                    features:=features.ToImmutableDictionary(Function(feature) feature, Function(feature) "true"))
```

Comment:
I'm putting the parsing into the refactoring of CompilationOptions.Features, as it fits a lot better into those changes than into here (in particular, Compilation.Feature was already parsing them, which that parsing routine should be shuffled around).